.github/workflows/release_please.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
         env:
           OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-        run: ./gradlew publishToSonatype closeSonatypeStagingRepository --no-daemon --stacktrace
+        run: ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository --no-daemon --stacktrace
```

Comment:
This makes sense based on my understanding of Sonatype. It may be why you were being asked for a staging ID before.